CHANGELOG.md

```diff
@@ -1,93 +1,175 @@
-## 1.9.0
-### Unreleased
+## 1.9.0-rc1
+### November 5, 2021
 
 CHANGES:
 
-* ui: replaces Vault's use of elazarl/go-bindata-assetfs in building the UI with Go's native Embed package [[GH-11208](https://github.com/hashicorp/vault/pull/11208)]
+* expiration: VAULT_16_REVOKE_PERMITPOOL environment variable has been removed. [[GH-12888](https://github.com/hashicorp/vault/pull/12888)]
+* expiration: VAULT_LEASE_USE_LEGACY_REVOCATION_STRATEGY environment variable has
+  been removed. [[GH-12888](https://github.com/hashicorp/vault/pull/12888)]
+* go: Update go version to 1.17.2
+* secrets/ssh: Roles with empty allowed_extensions will now forbid end-users
+  specifying extensions when requesting ssh key signing. Update roles setting
+  allowed_extensions to '*' to permit any extension to be specified by an end-user. [[GH-12847](https://github.com/hashicorp/vault/pull/12847)]
 
 FEATURES:
 
+* **Customizable HTTP Headers**: Add support to define custom HTTP headers for root path (`/`) and also on API endpoints (`/v1/*`) [[GH-12485](https://github.com/hashicorp/vault/pull/12485)]
+* **Elasticsearch in the UI**: Elasticsearch DB is now supported by the UI [[GH-12672](https://github.com/hashicorp/vault/pull/12672)]
```

Comment:
Can we make it more clear this is inside the database secrets engine?

Reply:
Any suggestions on structuring the message? There are a few that are like this.

Reply:
Updated in https://github.com/hashicorp/vault/pull/13067/commits/9cdb4f9f01da0fb623ae845f3295964eb5768e78. WDYT?